apps/vanilla/src/pages/books-app/books-app.css

```
@@ -0,0 +1,231 @@
+@import url("../../global/index.css");
+
+:root {
+  --desktop-cards-count: 5;
+  --desktop-card-width: 168px;
+  --desktop-card-height: 251px;
+  --desktop-card-border-radius: 24px;
+  --desktop-book-list-width:
+    calc(
+      (var(--desktop-card-width) * var(--desktop-cards-count)) +
+      (var(--double-gap) * var(--desktop-cards-count)
+    )
+  );
+}
+
+.header {
+  border-bottom: var(--separator-height) solid var(--separator-color);
+  position: fixed;
+  width: var(--desktop-width);
+  background: var(--background-color);
+  z-index: 99;
+}
+
+.typography__title {
+  color: var(--heading-color-1);
+  height: 64px;
+  line-height: 64px;
+  font-size: var(--font-size-lg);
+  font-weight: var(--bold-line-weight);
+  display: inline-block;
+  vertical-align: middle;
+}
+
+.typography__header {
+  width: var(--desktop-book-list-width);
+  background-color: var(--background-color);
+  color: var(--heading-color-3);
+  text-transform: uppercase;
+  font-size: var(--font-size-md);
+  font-weight: var(--semibold-line-weight);
+  padding: var(--triple-gap) 0;
+  word-spacing: 8px;
+}
+
+.typography__subtitle {
+  font-size: var(--font-size-sm);
+  font-weight: var(--regular-line-weight);
+  height: 48px;
+  position: sticky;
+}
+
+.books__year__header {
+  position: sticky;
+  top: 64px;
+  z-index: 1;
+}
+
+.main {
+  display: grid;
+  grid-template-columns: var(--desktop-book-list-width) 320px;
+  column-gap: var(--triple-gap);
+  padding-top: 64px;
+}
+
+.books__list__section {
+  padding-bottom: var(--triple-gap);
+}
+
+.books__list {
+  display: grid;
+  grid-template-columns:
+    var(--desktop-card-width)
+    var(--desktop-card-width)
+    var(--desktop-card-width)
+    var(--desktop-card-width)
+    var(--desktop-card-width);
+  gap: var(--double-gap);
+}
+
+.books__item {
+  width: var(--desktop-card-width);
+  position: relative;
+  overflow: hidden;
+}
+
+.books__item__image {
+  width: var(--desktop-card-width);
+  height: var(--desktop-card-height);
```

Image will preserve its aspect ratio. 

```suggestion
  width: 100%;
```

---

But then it will be with different height. And in other sizes we need another width, narrower

---

I updated it, please check